examples/employee-app/src/main/java/com/example/helidon/employee/EmployeeMockList.java

```diff
@@ -0,0 +1,72 @@
+/* Copyright 2019 (c) Oracle and/or its affiliates. All rights reserved. */
+package com.example.helidon.employee;
+
+import java.util.UUID;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+import javax.json.bind.JsonbConfig;
+
+public class EmployeeMockList {
+
+	private static final CopyOnWriteArrayList<Employee> eList = new CopyOnWriteArrayList<Employee>();
+	
+    static {
+    	JsonbConfig config = new JsonbConfig().withFormatting(Boolean.TRUE);
+
+    	Jsonb jsonb = JsonbBuilder.create(config);
+    	
+        String jsonString = "[{\"id\":\"" + "100" +"\",\"firstName\":\"Hugh\",\"lastName\":\"Jast\",\"email\":\"Hugh.Jast@example.com\",\"phone\":\"730-715-4446\",\"birthDate\":\"1970-11-28T08:28:48.078Z\",\"title\":\"National Data Strategist\",\"department\":\"Mobility\"},"

```

Comment:
It seems more practical to include the JSON as a resource file (src/main/resources) and loading it as a class-loader resource.

Reply:
i.e create a file under `src/main/resources/io/helidon/service/employee/employees.json`,
and then do `jsonb.fromJson(EmployeeMockList.class.getResourceAsStream("employees.json", new CopyOnWriteArrayList<Employee>(){}.getClass().getGenericSuperclass()))`

Reply:
This should be fixed now.